src/plugins/orangehrmAdminPlugin/test/fixtures/testcases/I18NLanguagesAPITestCase.yml

```diff
@@ -52,3 +85,44 @@ Update:
         id: 1
         name: 'Chinese (Simplified, China) - 中文（简体，中国）'
         code: 'zh_Hans_CN'
+
+Delete:
+  'with no ids':
+    userId: 1
+    services:
+      admin.localization_service: OrangeHRM\Admin\Service\LocalizationService
+    body:
+      ids: [ ]
+    data: [ ]
+
+  'with one id':
+    userId: 1
+    services:
+      admin.localization_service: OrangeHRM\Admin\Service\LocalizationService
+    body:
+      ids: [ 2 ]
+    data: [ 2 ]
+
+  'with multiple ids':
+    userId: 1
+    services:
+      admin.localization_service: OrangeHRM\Admin\Service\LocalizationService
+    body:
+      ids: [ 1, 2 ]
+    data: [ 1, 2 ]
+
+  'without int array':
+    userId: 1
+    services:
+      admin.localization_service: OrangeHRM\Admin\Service\LocalizationService
+    body:
+      ids: 1
+    invalidOnly: [ 'ids' ]
+
+  'with invalid ids':
+    userId: 1
+    services:
+      admin.localization_service: OrangeHRM\Admin\Service\LocalizationService
+    body:
+      ids: [ 1, "a" ]
+    invalidOnly: [ 'ids' ]
```

Comment:
Add a new line at the end. You can make sure this happens automatically by turning on the option PHPStorm settings > Editor > General

![image](https://github.com/orangehrm/orangehrm/assets/97493184/4808f3f1-b04b-4c0c-90c9-98dc82ce9b76)

Reply:
Done.